benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadServer.java

```diff
@@ -165,7 +165,7 @@
     }
   }
 
-  Executor getExecutor(int asyncThreads) {
+  ExecutorService getExecutor(int asyncThreads) {
```

Comment:
I was going to shutdown the executor after the benchmark was done in a followup CL